apps/bluetooth-music/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "bluetooth-music",
+  "version": "0.0.1",
+  "description": "Pre-installed skill for bluetooth music.",
+  "main": "app.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "manifest": {
+    "hosts": [
+      "bluetooth-music"
```

Comment:
All hosts should be listed.

Reply:
This app is only for bluetooth music player, so only `bluetooth-music` host can be write here.